PhysicsTools/PatAlgos/interface/PATUserDataMerger.h

```diff
@@ -99,11 +100,21 @@ namespace pat {
 // Constructor: Initilize user data src
 template<typename ObjectType, typename Operation>
 pat::PATUserDataMerger<ObjectType, Operation>::PATUserDataMerger(const edm::ParameterSet & iConfig, edm::ConsumesCollector & iC) :
-  userDataSrc_(iConfig.getParameter<std::vector<edm::InputTag> >("src"))
+  userDataSrc_(iConfig.getParameter<std::vector<edm::InputTag> >("src")),
+  labelPostfixesToStrip_(iConfig.existsAs<std::vector<std::string>>("labelPostfixesToStrip") ? iConfig.getParameter<std::vector<std::string>>("labelPostfixesToStrip") : std::vector<std::string>())
```

Comment:
looks like in this case fillDesciptions is not enough, maybe because the parent class doesn't percolate the description filling to this helper.